tests/selenium/popup_test.py

```diff
@@ -244,6 +247,42 @@ def test_donate_button(self):
         self.assertEqual(self.driver.current_url, EFF_URL,
             "EFF website should open after clicking donate button on popup")
 
+    def test_not_show_instruction(self):
+        """Ensure the instruction message doesn't display on firstRun.html ."""
+        self.open_popup(new_url=False)
+
+        try:
+            instruction = self.driver.find_element_by_id("instruction")
+        except NoSuchElementException:
+            self.fail("Unable to find instruction on popup")
+        self.assertEqual(instruction.value_of_css_property("display"), "none",
```

Comment:
Yep, this doesn't work because we don't have a proper way to open the popup on/for a particular page in tests (https://github.com/EFForg/privacybadger/issues/1634#issuecomment-338265211). Instead the popup is opened in the context of the popup itself I think. Why did this test pass some of the time in https://travis-ci.org/EFForg/privacybadger/builds/295583717?